config/manager/statefulset.yaml

```diff
@@ -12,6 +12,7 @@ spec:
     matchLabels:
       control-plane: controller-manager
       controller-tools.k8s.io: "1.0"
+  serviceName: controller-manager-service
```

Comment:
Service name is openstack-provider-controller-manager-service.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/config/manager/service.yaml#L7